packages/calcite-design-tokens/src/component/[template-comp-name].json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Is there a reason this filename needs to contain brackets? If not can you rename it so it doesn't, e.g `TEMPLATE.json` or even `_TEMPLATE.json`. Putting special characters (besides `-` and `_`) in filenames can lead to issues.

Reply:
the designers needed this file when they were using TokenStudio but we can remove it now.